test/unit/helpers/xhr/MFA_ENROLL_totp_success.js

```diff
@@ -32,7 +32,7 @@ define({
             "factorResult": "WAITING",
             "_links": {
               "qrcode": {
-                "href": "../../../test/assets/1x1.gif",
+                "href": "../../../test/unit/assets/1x1.gif",
```

Comment:
As discussed, might do this later depending on what else I need to change (casing issues on osx, etc)